### DISCUSSION TITLE
Anchoring a boat, create a zone dinamically from GPS position by the value of 2 sensors, and radius by an input_number

### DISCUSSION THREAD

**Discussion by Klagio** [frenck/spook#577] [Ideas]
Hi, can't see if this is already possible.

I wish to create a Zone (which will be a temporary Zone), based on trhe current object coordinates (in my case setting an anchor of a boat), and a radius based on a user defined input_number (case by case).

If the anchor gets dragged by the currents and the boat gets out of the radius, an automation should trigger an alarm.

I can achjieve this by putting a button on the dashboard, an input_number (the radius), when the button is pressed (anchor is set): when the buitton is pressed  a new Zone is created (but it has to have the GPS coordinates of the sensor of the time I press the button )and an alarm automation will be started,

**Comment by tjorim** [comment]
You can already create or update a zone with your own coordinates. ![Screenshot_20240208-101134.png](https://github.com/frenck/spook/assets/1294876/f5fcfbb7-9362-4a8d-89ca-53c9cea179be)

I would recommend that you don't always delete the zone but just have one that's called zone.boat and update it. The script that calls the spook service could also just enable an automation. Spook is only one part of your desired solution.

Edit: oh, I think I get it, it seems like the service only allows for numbers and not entities (attributes).
